### PR TITLE
Update hierarchy comps

### PR DESCRIPTION
**Pull request by trainorpj**
This is related to #139 

This PR does the following (each statement applies to both `<Tree />` and `<Cluster />` components:

1. Adds `className` to each `<Group />` component.
* Is this where the `className` prop should go?  I didn't see any other reasonable place to put it.
2. Adds `<DefaultLink />` and `<DefaultNode />` components to each component, along with basic tests for each.
* These render just fine--let me know if there are issues with them.
* I don't know much about testing components, so let me know if we need more comprehensive tests right now 🃏 

This PR **does not**

3. Add defaults to the `size`, `nodeSize`, and `separation` props.
* The defaults are set in the corresponding d3 functions.   The questions is, are we setting defaults at the highest or lowest level they're called at?  
  * For example, there is no default for the `top` prop in either component--the prop gets passed to `<Group />` which **does** have a default for `top`.  So the default is set there (that's what I mean by "lowest level". I think this is the best way to do it.
4. Update the Readme.  
* I can do this once the above points are addressed (let me know and I can add them before you merge 👍)

This is my first contribution to the codebase outside of documentation, so let me know if there's something I missed.  For example, it doesn't have today's commits to master (I probably need to do that, right? 😬).

### CODE REVIEW

**Comment by hshoff** [comment]
Looks good to me!

File: packages/vx-hierarchy/src/hierarchies/Cluster.js [hunk 1]
```diff
@@ -23,21 +25,23 @@ export default function Cluster({
...
-        );
-      })}
-      {nodeComponent && descendants.map((node, i) => {
-        return (
-          <Group key={`cluster-node-${i}`}>
-            {React.createElement(nodeComponent, { node })}
-          </Group>
-        );
-      })}
+    <Group top={top} left={left} className={className}>
```
Comment (hshoff) [inline packages/vx-hierarchy/src/hierarchies/Cluster.js]: let's change this to

```js
className={cx('vx-cluster', className)}
```

that way it has a default class and folks can add their own

File: packages/vx-hierarchy/src/hierarchies/Tree.js [hunk 2]
```diff
@@ -23,21 +25,23 @@ export default function Tree({
...
-        );
-      })}
-      {nodeComponent && descendants.map((node, i) => {
-        return (
-          <Group key={`tree-node-${i}`}>
-            {React.createElement(nodeComponent, { node })}
-          </Group>
-        );
-      })}
+    <Group top={top} left={left} className={className}>
```
Comment (hshoff) [inline packages/vx-hierarchy/src/hierarchies/Tree.js]: same here

```js
className={cx('vx-tree', className)}
```

File: packages/vx-hierarchy/src/hierarchies/Cluster.js (hunk 1 above)
Comment (trainorpj) [reply packages/vx-hierarchy/src/hierarchies/Cluster.js]: Got them both.  I also found a (small) bug in my code, adding className where it has no business being 🐛

**Comment by trainorpj** [comment]
I'll update the readme when this is merged (I need to have working links to the components).

**Pull request merged**